cytosnake.yaml

```diff
@@ -3,12 +3,13 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - conda-forge::python==3.10
+  - conda-forge::python=3.10.*
   - conda-forge::pip
-  - bioconda::snakemake
   - conda-forge::mamba
   - conda-forge::pre-commit
   - conda-forge::myst-parser
+  - conda-forge::pyyaml>=6.0.0
+  - bioconda::snakemake
   - pip:
     - rst-to-myst
-    - furo
+    - furo==2023.7.26
```

Comment:
```suggestion
    - furo==2023.7.26

```

Reply:
also, doesn't furo and myst belong in a `docs/requirements.txt` file?

Reply:
rst-to-myst is required for development, you are correct with furo!